src/core/shared_state/currentTooltip.ts

```diff
@@ -1,7 +1,12 @@
 import { createAtom } from '~utils/atoms';
 
-export type Position = 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
-export type Coords = { x: number; y: number; predefinedPosition?: Position };
+export type TooltipCorner = 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
+export type Coords = {
+  x: number;
+  y: number;
+  // TODO: rename to `corner`
+  predefinedPosition?: TooltipCorner;
```

Comment:
we already have prototypes where tooltips have something like 'center-top' position so i'm afraid corners if not enough

Reply:
Agree with @IlyaIzr, but adding more positions lays out this task scope.
My namings suggestion 
```ts
export type Position = {
  x: number;
  y: number;
  placement?: TooltipPlacement;
};
```
In the future, we will be able to add more placements if needed

Reply:
Another option - decouple `Coords` and `Position` to two separate `TooltipData` options